packages/terra-tabs/src/Tabs.module.scss

```diff
@@ -5,6 +5,14 @@
 @import './orion-fusion-theme/Tabs.module';
 
 :local {
+  .active-tab-icon {  
+    margin-right: var(--terra-tabs-icon-margin-right, 0.5rem);
+
+    svg {
+      font-size: var(--terra-tabs-icon-font-size, 1rem) !important;
+    }
+  }
```

Comment:
1. Terra doesn't use `!important` but rather increases specificity to the appropriate amount to never need it. Move this new block into the `.tab-menu` class block currently at line 85, after the `span` and `svg` declaration, remove the important and update the css variables:

```diff
  .tab-menu {
    align-items: baseline;
    display: flex;
    flex: 1 1 auto;
    justify-content: center;
    min-width: 7.143rem;

    span {
      flex: 0 1 auto;
      min-width: 0;
    }

    svg {
      flex: 0 0 auto;
      font-size: 0.571rem;
      margin-left: var(--terra-tabs-menu-arrow-margin-left, 0.5rem);
    }

+    .active-tab-icon {  
-      margin-right: var(--terra-tabs-icon-margin-right, 0.5rem);
+      margin-right: var(--terra-tabs-menu-icon-margin-right, 0.5rem);
+      
+      /* stylelint-disable-next-line selector-max-compound-selectors */
+      svg {
-        font-size: var(--terra-tabs-icon-font-size, 1rem) !important;
+        font-size: var(--terra-tabs-menu-icon-font-size, 1rem);
+      }
+    }
  }

```
\
\
2. The new variables need to be included in the Fusion and Lowlight theme files as well, so add to both, use same values for lowlight and 0.25rem for Fusion for margin-right:

Fusion theme:
```
--terra-tabs-menu-icon-margin-right: 0.25rem;
--terra-tabs-menu-icon-font-size: 1rem;
```
Lowlight:
```
--terra-tabs-menu-icon-margin-right: 0.5rem;
--terra-tabs-menu-icon-font-size: 1rem;
```
----
\
\
\
3. We also need to update the disabled styles for tabs as well. In `StructuralTabs.module.scss`, I found some wrong variables (said hover when they weren't) and need to change these styles to work with the new icons and making the right things use opacity to dim them when disabled:

```diff
      &.is-disabled {
-       background-color: var(--terra-tabs-structural-disabled-hover-background-color, #fff);
+       background-color: var(--terra-tabs-structural-disabled-background-color, #fff);
        background-size: 0 0;
-       color: var(--terra-tabs-structural-disabled-hover-color, rgba(0, 0, 0, 0.25));
+       color: var(--terra-tabs-structural-disabled-color, rgba(28, 31, 33, 0.2));
        cursor: default;
-       opacity: var(--terra-tabs-structural-disabled-hover-opacity, 1);
+       opacity: var(--terra-tabs-structural-disabled-opacity, 1);
      }

      @media screen and (-ms-high-contrast: active), (-ms-high-contrast: none), (hover: hover) and (pointer: fine) {
        &.is-disabled:hover {
          background-color: var(--terra-tabs-structural-disabled-hover-background-color, #fff);
          background-size: 0 0;
-         color: var(--terra-tabs-structural-disabled-hover-color, rgba(0, 0, 0, 0.25));
+         color: var(--terra-tabs-structural-disabled-hover-color, rgba(28, 31, 33, 0.2));
          cursor: default;
          opacity: var(--terra-tabs-structural-disabled-hover-opacity, 1);
        }
      }


      &.is-disabled .label,
      &.is-disabled svg {
+       color: var(--terra-tabs-structural-disabled-label-color, rgba(28, 31, 33, 0.7));
-       opacity: var(--terra-tabs-structural-disabled-label-opacity, 1);
+       opacity: var(--terra-tabs-structural-disabled-label-opacity, 0.3);
      }
```
\
4. Then we need to update the themes as well:
\
Fusion theme:
```diff
+    --terra-tabs-structural-disabled-background-color: #fff;
+    --terra-tabs-structural-disabled-opacity: 1;
+    --terra-tabs-structural-disabled-color: rgba(28, 31, 33, 0.3);
     --terra-tabs-structural-disabled-hover-background-color: #fff;
     --terra-tabs-structural-disabled-hover-opacity: 1;
-    --terra-tabs-structural-disabled-hover-color: #9b9fa1;
+    --terra-tabs-structural-disabled-hover-color: rgba(28, 31, 33, 0.3);

+    --terra-tabs-structural-disabled-label-color: #1c1f21;
-    --terra-tabs-structural-disabled-label-opacity: 1;
+    --terra-tabs-structural-disabled-label-opacity: 0.3;

```

Lowlight theme:
```diff
    --terra-tabs-structural-content-box-shadow: 0;
+   --terra-tabs-structural-disabled-label-color: #b2b5b6;
-   --terra-tabs-structural-disabled-label-opacity: 1;
+   --terra-tabs-structural-disabled-label-opacity: 0.3;
    --terra-tabs-structural-content-border-radius: 0;
    --terra-tabs-structural-content-z-index: 0;

...

+   --terra-tabs-structural-disabled-background-color: #222a2e;
+   --terra-tabs-structural-disabled-color: #404344;
+   --terra-tabs-structural-disabled-opacity: 1;
    --terra-tabs-structural-disabled-hover-background-color: #222a2e;
    --terra-tabs-structural-disabled-hover-color: #404344;
    --terra-tabs-structural-disabled-hover-opacity: 1;
```

Reply:
done:  https://github.com/cerner/terra-framework/pull/1592/commits/361acc96bc2d62d7781db702311d07c9321d52ba